config/cluster/kustomization.yaml

```diff
@@ -0,0 +1,38 @@
+# Set WATCH_NAMESPACE to "" in the deployment to provide cluster-wide install
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    name: k8ssandra-operator
+    kind: Deployment
+  patch: |-
+    - op: remove
+      path: /spec/template/spec/containers/0/env/0/valueFrom
+    - op: add
+      path: /spec/template/spec/containers/0/env/0/value
+      value: ""
+- target:
+    group: rbac.authorization.k8s.io
+    version: v1
+    kind: Role
+    name: k8ssandra-operator
+  patch: |-
+    - op: replace
+      path: /kind
+      value: ClusterRole
+- target:
+    group: rbac.authorization.k8s.io
+    version: v1
+    kind: RoleBinding
+    name: k8ssandra-operator
+  patch: |-
+    - op: replace
+      path: /roleRef/kind
+      value: ClusterRole
+    - op: replace
+      path: /kind
+      value: ClusterRoleBinding
+resources:
+- ../../config/default
```

Comment:
You cannot use `../../config/default` because it will deploy the namespace-scoped cass-operator. I pushed changes to my branch. See https://github.com/jsanda/k8ssandra-operator/commit/bf6d4fd338313cf8cfa3c8653fdf8042ad5a1237 to address this. I introduced another kustomization directory because I assume we want k8ssandra-operator deployed in a namespace of the same name and the same for cass-operator.